ws-nextjs-app/pages/[service]/new_live/[id]/[[...variant]].page.tsx

```diff
@@ -140,6 +140,7 @@ export const getServerSideProps: GetServerSideProps = async context => {
       timeOnServer: Date.now(), // TODO: check if needed?
       toggles,
       variant: variant?.[0] || null,
+      isUK: reqHeaders['x-bbc-edge-isuk'] === 'yes',
```

Comment:
It seems that this is duplicated here and in ws-nextjs-app/pages/[service]/[[...]].page.tsx - is it worth adding a reusable function and using it in both places?